include/fuzzy/fuzzy_match.hh

```diff
@@ -50,27 +122,39 @@ namespace fuzzy
                float fuzzy,
                unsigned number_of_matches,
                std::vector<Match> &matches,
+               float contrastive_factor=0,
```

Comment:
Can you move all new arguments at the end of the list, so that the usage is unchanged for existing users?